modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,98 @@
+
+* `_(INF)_` allows Infantry units to use the {portal}.
+* `_(4+)_` allows Vehicle units that have an Armour value of `4+` or lower to use the {portal}.
+* `_(Walker)_` allows Vehicle units that have the {walker} ability to use the {portal}, regardless of their Armour value.
+
+We also indicate how the portal affects the arrival of detachments that use it:
+
+* `_(Auto)_` means that detachments in Portal Reserves do *not* follow the standard rules for Reserves.
+Instead, from turn two onwards, you must *must* disembark them from a {portal} unit/objective in the Movement phase unless there are not enough portals available, or none of the available portals can disembark all of the units in the detachment.
+* `_(Roll)_` means that detachments in Portal Reserves follow the standard rules for Reserves.
+When a detachment arrives from Portal Reserves, you *must* disembark it from a {portal} unit/objective, if possible.
+
+== Game effects of {portal}
+
+Where your army includes any units or objectives that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that can use a portal out of the play area, as <<Portal Reserves>>.
+
+Where your unit or objective has the {portal} ability:
+
+* When you wish to move the units in a detachment out of a portal, use the normal disembarkation rules as at xref:main-rules:transportation.adoc[].
+footnote:[
+When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
+In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
+]
+* When you wish to embark or disembark a detachment through a portal, you may choose only Advance or Assault orders.
+* You may use each individual portal only *once per turn*, to embark or disembark a single detachment.
+* In the Movement phase, you may embark friendly units 'into' a portal.
+But then you *must* immediately disembark them from a different {portal} elsewhere in the play area -- you may not place them outside of the play area again, so they cannot become Portal Reserves again.
+
+== Portal Reserves
+
+Apply these rules to each detachment that you have outside of the play area as Portal Reserves.
+
+When the detachment should arrive, you must put it into play if possible -- but you may only put it into play through a suitable, available {portal}.
+
+* If there is no portal available when you need it, then you must leave the detachment in Portal Reserves this turn -- it does not arrive yet after all.
+You must roll again next turn.
+* If none of the available portals can disembark all of the units in the detachment, then you may choose, either:
+** Leave the detachment in Portal Reserves this turn -- it does not arrive yet after all.
+You must roll again next turn.
```

Comment:
Confusing -- the roll isn't relevant where _`(Auto)`_ is in effect, but it's also unclear how this actually _goes_ into effect alongside _`(Roll)`_.

Reply:
This appears to have got a bit garbled during editing. 

I had two separate bullet points, one for _`(Roll)`_ and one for _`(Auto)`_, so 'roll' was never mentioned for _`(Auto)`_.

Reply:
The Roll bullet said:

* If the {portal} lists 'Roll' in brackets, detachments in {portal-reserves} follow the standard rules for Reserves. When a detachment arrives from {portal-reserves}, it **must** disembark from a {portal}. If there are not enough Portals available, or none of the available Portals can disembark all the units in the detachment, the detachment stays in {portal-reserves} and rolls again next turn.

Reply:
Got it — now that I understand from your other comment that these attributes are mutually exclusive, I'll be able to figure out what to do with the edits.